BikeRentalApi/endpoints/malfunctionsList.py

```diff
@@ -0,0 +1,21 @@
+from django.http import JsonResponse
+from rest_framework import status
+
+from BikeRentalApi.decorators.roleRequired import RoleRequired
+from BikeRentalApi.enums import Role
+from BikeRentalApi.models import Malfunction
+# POST: create bikes
+# GET: list all bikes
```

Comment:
I don't think these comments belong here. Please remove them or change them and put them in line 1

Reply:
Fixed